src/EFCore.MySql/Query/Internal/MySqlStringMethodTranslator.cs

```diff
@@ -50,6 +52,12 @@ public class MySqlStringMethodTranslator : IMethodCallTranslator
             = typeof(string).GetRuntimeMethod(nameof(string.Contains), new[] { typeof(string) });
         private static readonly MethodInfo _endsWithMethodInfo
             = typeof(string).GetRuntimeMethod(nameof(string.EndsWith), new[] { typeof(string) });
+        private static readonly MethodInfo _matchMethodInfo
+            = typeof(StringExt).GetRuntimeMethod(nameof(StringExt.Match), new[] {typeof(string), typeof(string)});
+        private static readonly MethodInfo _matchInBooleanModeMethodInfo
+            = typeof(StringExt).GetRuntimeMethod(nameof(StringExt.MatchInBooleanMode), new[] {typeof(string), typeof(string)});
+        private static readonly MethodInfo _matchWithQueryExpansionMethodInfo
+            = typeof(StringExt).GetRuntimeMethod(nameof(StringExt.MatchWithQueryExpansion), new[] {typeof(string), typeof(string)});
```

Comment:
I am not convinced that we should add a `Match()` string extension method. I think we should handle this similar to `LIKE`, that just has a `DbFunctions` extension method. So I think this should be removed.

If we decide to include this anyway, this should not be in `MySqlStringMethodTranslator`, but in a new method translator class.